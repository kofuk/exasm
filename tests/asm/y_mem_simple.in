sw r3, (r5)
lw r1, (r7)
sbu r3, (r5)
lbu r1, (r7)
