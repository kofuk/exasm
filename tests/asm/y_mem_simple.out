@00 00000011 10110000 // sw r3, (r5)
@02 00000001 11110001 // lw r1, (r7)
@04 00000011 10110010 // sbu r3, (r5)
@06 00000001 11110011 // lbu r1, (r7)
